### PR TITLE
Add detach option to session

### PR DESCRIPTION
**Pull request by LuKks**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ module.exports = class Corestore extends ReadyResource {
     this._replicationStreams = root ? root._replicationStreams : []
     this._overwrite = opts.overwrite === true
     this._readonly = opts.writable === false
+    this._attached = opts._attached || null
```
Comment (HDegroote) [inline index.js:39]: Detail: `_attached` is internal-only, so I don't think the `||null` is needed (the check in close is cast to bool anyway, so can be anything falsy, but that's irrelevant given that you're passing in null or an object)

Comment (mafintosh) [reply index.js:39]: It should be || null as its part of the initial opts passed by the user. Any prop stored on a class should NEVER be undefined, as that makes debugging much harder than it should be

Comment (HDegroote) [reply index.js:39]: AH yes, my bad, hadn't considered the flow for the original store.

**Comment by mafintosh** [comment]
6.11.0

**Pull request merged**